maintainer/cuda_build.sh

```diff
@@ -0,0 +1,31 @@
+GIT_COMMIT=$(git rev-parse HEAD)
```

Comment:
That is even very important!  The `$(...)` command execution is Bash specific.  POSIX only mandates `` `...` ``.